gwt/pom.xml

```diff
@@ -40,39 +40,10 @@
       <classifier>sources</classifier>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>javax.inject</groupId>
+      <artifactId>javax.inject</artifactId>
+      <classifier>sources</classifier>
```

Comment:
Yes, that's brought transitively (dependencies are transitive by default, one has to mark them optional or provided to make them non-transitive, or explicitly exclude them if you don't want them)